src/supla/network/html/button_update.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SRC_SUPLA_NETWORK_HTML_BUTTON_UPDATE_H_
+#define SRC_SUPLA_NETWORK_HTML_BUTTON_UPDATE_H_
+
+#include <supla/network/html_element.h>
+#include <supla/network/esp_web_server.h>
+
+namespace Supla {
+namespace Html {
+
+class ButtonUpdate : public HtmlElement {
+ private:
+  Supla::EspWebServer* server;  // Wskaźnik na serwer
```

Comment:
use English in comments.
Add here: = nullptr; - so it will be initialized by default to nullptr.